src/octoprint/plugins/action_command_notification/__init__.py

```diff
@@ -130,6 +139,13 @@ def _clear_notifications(self):
         self._plugin_manager.send_plugin_message(self._identifier, {})
         self._logger.info("Notifications cleared")
 
+    def _configure_filters(self):
+        filters = self._settings.get(["filters"]).split("\n")
+
+        self._filters.clear()
+        for filter in filters:
+            self._filters.append(re.compile(filter))
```

Comment:
This gives you a brief period where the filters are completely unset. If we didn't have to think about error handling here, I'd suggest to go for a more atomic (and also IMHO more pythonic) approach like this:

```suggestion
        self._filters = [re.compile(filter) for filter in filters]
```

However, a broken regex might be configured by the user and throw an exception here during compile, that needs to be handled gracefully. 

In any case, as suggested above, we should only end up with *one* regex in the backend.